pkg/types/indicator.go

```diff
@@ -112,6 +112,7 @@ type SeriesExtend interface {
 	Softmax(window int) SeriesExtend
 	Entropy(window int) float64
 	CrossEntropy(b Series, window int) float64
+	Filter(b func(i int, value float64) bool, length int) SeriesExtend
```

Comment:
can to add a doc comment for this Filter method?

Reply:
for example, how does it do filtering?

Reply:
added in filter.go